lib/developer_portal/app/controllers/developer_portal/admin/account/personal_details_controller.rb

```diff
@@ -61,4 +63,17 @@ def redirect_path
     end
   end
 
+  def user_params
+    params.require(:user)
+  end
+
+  def current_password_verification
+    return true unless current_user.using_password?
```

Comment:
Preemptive review (2 in 1 bargain edition):
1. Changing the name of the method to an action makes its intentions clearer.
2. Since you're not using the `true` value this method return, if can simply return `nil`. Also, if there's a `return true`, I'd also expect the possibility of a `return false`. 

```suggestion
  def verify_current_password
    return unless current_user.using_password?
```